augur/index.py

```diff
@@ -0,0 +1,73 @@
+"""
+Count occurrence of bases in a set of sequences
+"""
+
+from Bio import SeqIO
+import csv
+
+def register_arguments(parser):
+    parser.add_argument('--sequences', '-s', required=True, help="sequences in fasta or VCF format")
+    parser.add_argument('--output', '-o', help="output file", required=True)
+
+
+def run(args):
+    '''
+    counts the number of a,c,t,g,n and other IUPAC nucleotides in a set of sequences
+    '''
+    #read in files
+    try:
+        seqs = SeqIO.parse(args.sequences, 'fasta')
+    except ValueError as error:
+        print("ERROR: Problem reading in {}:".format(args.sequences))
+        print(error)
+        return 1
+
+    good_chars = {'a', 'c', 'g', 't', 'n', '-', 'r', 'y', 's', 'w', 'k', 'm', 'd', 'h', 'b', 'v','?'}
```

Comment:
Why don't you construct `good_chars` from the `values`. Then one wouldn't need to keep to separate sets up-to-date.